bodhi/server/models.py

```diff
@@ -789,6 +789,14 @@ class Update(Base):
     user_id = Column(Integer, ForeignKey('users.id'))
 
     @property
+    def mandatory_days_in_testing(self):
+        if self.critpath:
+            return int(config.get('critpath.stable_after_days_without_negative_karma'))
+
+        days = self.release.mandatory_days_in_testing
+        return days if days else 0
```

Comment:
Hello @nbianca!

[```Release.mandatory_days_in_testing()```](https://github.com/fedora-infra/bodhi/blob/2.5.0/bodhi/server/models.py#L411-L424) does something that is admittedly a bit convoluted in that it loads a dynamically selected config key to determine what the value should be (instead of getting it from the DB or the object).

Thus, you could use [```mock.patch.dict()```](http://www.voidspace.org.uk/python/mock/patch.html#patch-dict) to patch in a config item for a given release. I've not tested this, but something along these lines should work:

```
@mock.patch.dict('bodhi.server.models.config', {'fc10.status': 'stable'})
@mock.patch.dict('bodhi.server.models.config', {'fc10.stable.mandatory_days_in_testing': '0'})
def test_blah_blah_blah(self):
    # Code in here should see 0 as the mandatory days in testing
```

By the way I didn't realize it worked this way when I wrote my previous comment but since it's using ```int()``` on the value you won't be able to encode ```None```, so let's use ```'0'``` as the "falsy" value.